cmd/mailchain/internal/http/handlers/resolve_name.go

```diff
@@ -60,6 +60,12 @@ func GetResolveName(resolvers map[string]nameservice.ForwardLookup) func(w http.
 			errs.JSONWriter(w, http.StatusNotAcceptable, errors.Errorf("%q not supported", protocol+"/"+network))
 			return
 		}
+		if nameservice.IsRfc1035Error(err) {
```

Comment:
this will be the only `nameservice.Is....` function needed as the later comments explain how to remove the other functions